templates/prSignUp.html

```diff
@@ -1,33 +1,32 @@
 <!DOCTYPE html>
```

Comment:
Are changes in this file relevant to the bug you found?

Reply:
No the changes on the right seem to be the original version of the code we edited. It's not relevant at all.

I don't know how it reverted the changes, (maybe they were stashed or something) but we don't want the code in. green.